kolibri_explore_plugin/assets/src/views/ContentItem.vue

```diff
@@ -73,6 +73,10 @@
         type: Object,
         required: true,
       },
+      dark: {
```

Comment:
We can use a more meaningful property name

```suggestion
      isDark: {
```